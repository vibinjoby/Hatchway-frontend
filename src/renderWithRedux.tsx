import { render } from '@testing-library/react';
import { Provider } from 'react-redux';

import { store } from './store/store';

const renderWithRedux = (component: any) => {
  return {
    ...render(<Provider store={store}>{component}</Provider>),
  };
};

export default renderWithRedux;
