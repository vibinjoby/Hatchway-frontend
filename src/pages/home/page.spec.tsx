import '@testing-library/jest-dom/extend-expect';
import Home from './index';
import renderWithRedux from '../../renderWithRedux';

test('renders a message', () => {
  const { container } = renderWithRedux(<Home />);
  expect(container.firstChild).toMatchSnapshot();
});
