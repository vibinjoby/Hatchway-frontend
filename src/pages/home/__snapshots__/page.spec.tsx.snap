// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a message 1`] = `
<div
  class="sc-fujyAs hipAGk"
>
  <div
    class="sc-pNWdM kOgDLe"
  >
    <div
      class="MuiPaper-root MuiCard-root card MuiPaper-elevation1 MuiPaper-rounded"
    >
      <div
        class="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root-1 name-search"
      >
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiInput-input"
            id="name-input"
            placeholder="Search by name"
            type="text"
            value=""
          />
        </div>
      </div>
      <div
        class="MuiFormControl-root MuiTextField-root WithStyles(ForwardRef(TextField))-root-1 tag-search"
      >
        <div
          class="MuiInputBase-root MuiInput-root MuiInput-underline MuiInputBase-formControl MuiInput-formControl"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiInput-input"
            id="tag-input"
            placeholder="Search by tag"
            type="text"
            value=""
          />
        </div>
      </div>
      <nav
        class="MuiList-root card-list MuiList-padding"
      />
    </div>
  </div>
</div>
`;
